packages/typespec-ts/test/integration/generated/azure/resource-manager/models/resources/src/index.d.ts

```diff
@@ -10,6 +10,44 @@ import { RawHttpHeaders } from '@azure/core-rest-pipeline';
 import { RequestParameters } from '@azure-rest/core-client';
 import { StreamableMethod } from '@azure-rest/core-client';
 
+export declare interface AccessRule {
```

Comment:
do you know why we have this change?

Reply:
https://github.com/Azure/typespec-azure/pull/1505/files#diff-ea8952908645379e09c4bb5179b9f2ec2d1105b261ac49cd8a17d73040094e9f

Reply:
why the change in the resource-manager lib would impact the result here?

Reply:
this is because RLC would generate child classes even it is not referenced.